app/views/index.erb

```diff
@@ -0,0 +1,24 @@
+<center>
+ <h1>Chitter</h1>
+<% if @current_user %>
+  Welcome, <%= @current_user.username %>
+  <% else %>
+  <a href='/users/new' id='sign-up' class='button'>Join Chitter</a>
+  <a href='/sessions/new' id='sign-in' class='button'>Sign in</a>
+  <% end %>
+<ul>
+<% if @peeps %>
+  <% @peeps.each do |peep| %>
+    <% @peeps %>
+    <li><%= peep.post%></li>
+  <% end %>
+</ul>
+<% end %>
+<div id="new-peep">
+  <form action="/peeps" method="post">
+    <li><p>Create Peep: <input type="text" size="50" name="post"></p></li>
+    <li><p>Username: <input type="text" name="user_name"></p></li>
```

Comment:
kevin; sorry I didn't immediately follow up on this - end of the day and all that - looking over it I think @spike01 's point is that you shouldn't ask the user to specify who creates the peep - you should set the user who is currently logged in as the user associated with the peep automatically, and BTW, do ask follow up questions within the pull request - it's the developer's way!